python/ray/tests/test_basic_2.py

```diff
@@ -673,6 +675,15 @@ def test_internal_config_when_connecting(ray_start_cluster):
         ray.get(oid)
 
 
+def test_get_correct_node_ip():
+    with patch("ray.worker") as worker_mock:
+        node_mock = MagicMock()
+        node_mock.node_ip_address = "10.0.0.111"
+        worker_mock._global_node = node_mock
+        found_ip = ray.services.get_node_ip_address()
+        assert found_ip == "10.0.0.111"
```

Comment:
```suggestion
        assert found_ip == "10.0.0.111"
        assert "node:10.0.0.111" in ray.cluster_resources()
        
```

Reply:
thanks for the advice.

Reply:
Hi @richardliaw, here can't `ray.cluster_resources` because it need ray initialized.